workflows/crisprseq_targeted.nf

```diff
@@ -148,14 +142,42 @@ workflow CRISPRSEQ_TARGETED {
 
     // Join channels with reference and protospacer
     // to channel: [ meta, reference, protospacer]
-    SEQ_TO_FILE_REF.out.file
-        .join(INPUT_CHECK.out.protospacer
-            .map {
-                meta, fastq ->
-                    [ meta - meta.subMap('id') + [id: meta.id.split('_')[0..-2].join('_')], fastq ]
-            },
-            by: 0)
-        .set{ reference_protospacer }
+    if (!params.reference_fasta && !params.protospacer) {
+        SEQ_TO_FILE_REF.out.file
+            .join(INPUT_CHECK.out.protospacer
+                .map {
+                    meta, fastq ->
+                        [ meta - meta.subMap('id') + [id: meta.id.split('_')[0..-2].join('_')], fastq ]
```

Comment:
When reading the input sample sheet, we add a `_T<number>` suffix to the sample name (see https://github.com/nf-core/crisprseq/blob/master/bin/check_samplesheet.py#L120). We use this to concatenate the fastq files if one sample has more than fastq file. Here we are removing this suffix so that the meta map in other channels matches the one of reads.